jlibra-core/src/main/java/dev/jlibra/admissioncontrol/query/GetTransactions.java

```diff
@@ -0,0 +1,23 @@
+package dev.jlibra.admissioncontrol.query;
+
+import org.immutables.value.Value;
+import types.GetWithProof;
+import types.GetWithProof.RequestItem;
+
+@Value.Immutable
+public abstract class GetTransactions {
+    public abstract long getStartVersion();
+    public abstract long getLimit();
+    public abstract boolean getFetchEvents();
+
+    public RequestItem toGrpcObject() {
```

Comment:
Some are `abstract class`es and some are `interface`s in this package. Should we clean this package a bit? What do you think?

Reply:
Right,I agree with you.Mixing up is a trouble.
 'GetAccountState.java' & 'GetAccountTransactionBySequenceNumber.java'  also need to be optimized.
Do you have any good optimization plans?

Reply:
I think this is ok for now, we currently have both interfaces and abstract classes, we could unify them as a separate chore. We have been talking about extracting the serialization / deserialization to separate classes but it's still a bit open how to do it. I think this implementation is in line with the existing classes.

Reply:
Well, so be it